x-pack/filebeat/module/cisco/ios/config/pipeline.js

```diff
@@ -215,8 +63,7 @@ var ciscoIOS = (function() {
             if (evt.Get("input.type") === "log") {
                 parseSyslogFileHeader.Run(evt);
             }
-
-            processMessage.Run(evt);
+            setECSCategorization(evt);
```

Comment:
My JS is not the best, but is this not running in a loop? Maybe @marc-gr could pitch in?